apiary.apib

```diff
@@ -163,6 +163,41 @@ FORMAT: 1A
               "message": "An email has been sent to 'user-5@example.com' containing instructions for resetting your password."
             }
 
+## Facebook [/auth/facebook/callback]
+
++ Attributes (object)
+  + access_token (required) - Access token provided by facebook
+
+### Login or Sign up [POST]
+
++ Request Ok
+
+    + Body
+
+            facebook[access_token]=acc35T0k3n
```

Comment:
?? is this param ok?

Reply:
Yes. It's what the client should send.

Reply:
I mean, why nested like that? and shouldn't that be a json?

Reply:
It would be json after I merge #47
And it's nested just to use strong parameters and require the facebook key and permit the access_token key.

Reply:
cool, thanks for the clarification 👍